frontend/src/components/sightingCard/sightingCard.tsx

```diff
@@ -0,0 +1,41 @@
+import "./sightingCard.scss";
+import { SightingResponse } from "../../clients/internalApiClient";
+import { format, parseISO } from "date-fns";
+
+interface SightingCardProps {
+  sighting: SightingResponse;
+}
+
+export function SightingCard(props: SightingCardProps): JSX.Element {
+  const sighting = props.sighting;
+  const species = props.sighting.species;
```

Comment:
Switching over to object destructuring as above will change these!
```suggestion
  const species = sighting.species;
```